src/main/java/teammates/ui/webapi/output/CommentSearchResultData.java

```diff
@@ -0,0 +1,15 @@
+package teammates.ui.webapi.output;
+
+import teammates.common.datatransfer.FeedbackSessionResultsBundle;
+
+/**
+ * API output format for a comment search result.
+ */
+public class CommentSearchResultData extends SessionResultsData {
+    private final FeedbackSessionData feedbackSession;
+
+    public CommentSearchResultData(FeedbackSessionResultsBundle bundle) {
```

Comment:
Please create a new `protected` constructor in SessionResultsData, you won't need `FeedbackSessionResultsBundle`

Reply:
In that case, would `CommentSearchResultData` even need a `FeedbackSessionResultsBundle` (as opposed to a different type to retrieve the `FeedbackSessionData`)?

Reply:
Why you need `FeedbackSessionResultsBundle` in the first place? Does `Logic#searchFeedbackResponseComments` gives you `FeedbackSessionResultsBundle`?

Reply:
Good point, let me take a closer look and define the constructor properly.